.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
```

Comment:
where did you get the limitation for tag from?

Reply:
Not a limitation. It won't trigger if you only push tags if you specify branches.